app/screens/home/channel_list/servers/servers_list/index.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {ListRenderItemInfo, StyleSheet, View} from 'react-native';
+import {FlatList} from 'react-native-gesture-handler';
+
+import {useServerUrl} from '@context/server';
+import {useIsTablet} from '@hooks/device';
+import BottomSheetContent from '@screens/bottom_sheet/content';
+
+import ServerItem from './server_item';
+
+import type ServersModel from '@typings/database/models/app/servers';
+
+type Props = {
+    servers: ServersModel[];
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+    },
+    contentContainer: {
+        marginVertical: 4,
+    },
+});
+
+const keyExtractor = (item: ServersModel) => item.url;
+
+const ServerList = ({servers}: Props) => {
+    const intl = useIntl();
+    const isTablet = useIsTablet();
+    const serverUrl = useServerUrl();
+
+    const onAddServer = useCallback(() => {
+        // eslint-disable-next-line no-console
```

Comment:
Is this functionality part of another ticket?  Should we add a todo with a ticket number here?

Reply:
ok